README.md

```diff
@@ -63,3 +70,80 @@ By default, the docker-compose file has ports set up such that:
 * The **Configuration Database** is accessible from <http://127.0.0.1:7000/> and its admin interface is accessible from <http://127.0.0.1:7000/admin/>.
 
 * The **Downtime Database** is accessible from <http://127.0.0.1:7500/> and its admin interface is accessible from <http://127.0.0.1:7500/admin/>.
+
+* The **Science Archive** should be accessible from <http://127.0.0.1:9500/> and its admin interface is accessible from <http://127.0.0.1:9500/admin/>.
+
+* The **minio bucket** which emulates S3 should be accessible from <http://127.0.0.1:9000/>
+
+## Manually adding data to the Science Archive
+
+The best way to add data to the science archive is by using the [ingester library](https://github.com/observatorycontrolsystem/ocs_ingester).
+A single file has been automatically been uploaded in the startup routine of the science archive, and is visible at the archive api: <http://127.0.0.1:9500/frames>.
+
+You can upload additional data as demonstrated in the following python script:
+
+```python
+import os
+import requests
+from ocs_ingester import ingester
+
+
+# The following environment variables are required by the ingester.
```

Comment:
I would just remove all this from the readme and replace it with a line referring to the ingester script file to see how the example file is ingested, and a line referring to the ingester github readme to see the environment variables that must be set to use the ingester library.